graphs/bipartite.py

```diff
@@ -35,6 +35,12 @@
 
 
 def bipartite(partition: list, adjacent_edges: list) -> int:
```

Comment:
```suggestion
def bipartite(partition: list, adjacent_edges: list) -> bool:
```
It would be more Pythonic to return `True` or `False` instead of 0 or 1